src/Microsoft.NetCore.Analyzers/Core/Runtime/InstantiateArgumentExceptionsCorrectly.cs

```diff
@@ -26,16 +25,36 @@ public sealed class InstantiateArgumentExceptionsCorrectlyAnalyzer : DiagnosticA
         private static readonly LocalizableString s_localizableMessageIncorrectParameterName = new LocalizableResourceString(nameof(MicrosoftNetCoreAnalyzersResources.InstantiateArgumentExceptionsCorrectlyMessageIncorrectParameterName), MicrosoftNetCoreAnalyzersResources.ResourceManager, typeof(MicrosoftNetCoreAnalyzersResources));
         private static readonly LocalizableString s_localizableDescription = new LocalizableResourceString(nameof(MicrosoftNetCoreAnalyzersResources.InstantiateArgumentExceptionsCorrectlyDescription), MicrosoftNetCoreAnalyzersResources.ResourceManager, typeof(MicrosoftNetCoreAnalyzersResources));
 
-        internal static DiagnosticDescriptor Descriptor = new DiagnosticDescriptor(RuleId,
+        internal static DiagnosticDescriptor RuleNoArguments = new DiagnosticDescriptor(RuleId,
                                                                              s_localizableTitle,
-                                                                             "{0}",
```

Comment:
We used to have only one descriptor that was using a `{0}` format. Instead, I have created 3 different descriptor to match what's done in other analyzers.